src/ontology/mondo-edit.obo

```diff
@@ -368936,6 +368936,9 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_subtype_of_a_disorder {source="Orphanet:206436"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "early onset Krabbe disease" EXACT []
+synonym: "early symptomatic Krabbe" EXACT []
+synonym: "early-onset Krabbe disease" EXACT []
```

Comment:
Please add a "database cross-reference" annotation to notify where this synonym comes from (ie where is this name used). 
This database x-ref can be a PMID, another x-ref (eg Orphanet:206436), and/or an orcid)